changelog.txt

```diff
@@ -27,6 +27,7 @@
 * Fix - Prevent empty strings being saved in related orders cache ID meta when backfilling order data to the WP Posts table.
 * Fix - Correctly load product names with HTML on the cart and checkout shipping rates.
 * Dev - Fix Node version mismatch between package.json and .nvmrc (both are now set to v16.17.1).
+* Dev - Removed support for WC version <= 4.4.
```

Comment:
I think we'll need to move this up to the top section.

Reply:
Fixed when merging trunk into the branch 766991d8ea675df182b6055a584591bb8b4564f3